website/docs/guides/orchestration/webhooks/zapier-slack.md

```diff
@@ -0,0 +1,304 @@
+
+Add another step, **Filter**. Set the **Field** to **2. Send Error Thread** and the **condition** to **(Boolean) Is true**. This prevents the Zap from failing if the job succeeded and you try to send an empty Slack message in the next step. 
+
+![Screenshot of the Zapier UI, showing the correctly configured Filter step](/img/guides/orchestration/webhooks/zapier-slack/filter-config.png)
+
+Add another **Send Channel Message in Slack** action. In the **Set up action** area, choose the same channel as last time, but set the **Message Text** to **2. Threaded Errors Post** from the same Run Python step. Set the **Thread** value to **3. Message Ts**, the timestamp of the post created by the first Slack action. This tells Zapier to add this post as a threaded reply to the main message, which prevents the full (potentially long) output from cluttering your channel. 
+
+![Screenshot of the Zapier UI, showing the mappings of prior steps to a Slack message](/img/guides/orchestration/webhooks/zapier-slack/thread-slack-config.png)
+
+### 7. Test and deploy
+
+When you're happy with your Zap, remember to ensure that your `run_id` and `account_id` are no longer hardcoded in the Code step, then publish your Zap.
+
+## Alternate approach
+
+Instead of using a webhook as your trigger, you can keep the existing dbt Cloud app installed in your Slack workspace and use its messages being posted to your channel as the trigger. In this case, you can skip validating the webhook and only need to load the context from the thread. 
+
+### 1. Create a new Zap in Zapier
+Use **Slack** as the initiating app, and **New Message Posted to Channel** as the Trigger. In the **Trigger** configuration area, pick the channel where your Slack alerts are posted, and set **Trigger for Bot Messages?** to **Yes**.
+
+![Screenshot of the Zapier UI, showing the correctly configured Message trigger step](/img/guides/orchestration/webhooks/zapier-slack/message-trigger-config.png)
+
+Test your Zap to find an example record. You may need to load additional samples until you get one which relates to a failed job, depending on whether you post all job events to Slack or not. 
+
+### 2. Add a Filter step
+Add a **Filter** step with the following conditions: 
+- **1. Text contains failed on Job**
+- **1. User Is Bot Is true**
+- **1. User Name Exactly matches dbt Cloud**
+
+![Screenshot of the Zapier UI, showing the correctly configured Filter step](/img/guides/orchestration/webhooks/zapier-slack/message-trigger-filter.png)
+
+### 3. Extract the Run ID 
+Add a **Format** step with the **Event** of **Text**, and the Action **Extract Number**. For the **Input**, select **1. Text**. 
+
+![Screenshot of the Zapier UI, showing the Transform step configured to extract a number from the Slack message's Text property](/img/guides/orchestration/webhooks/zapier-slack/extract-number.png)
+
```

Comment:
It does, but there are a LOT of places where I'm not saying "click continue" because each zap step has like 4 substeps that you move through and it gets very messy very quickly. Using "we assume some familiarity with Zapier" as my get out of jail card here 🙈

Reply:
i'm good with that! if we get support issues, we can always go in and update!